helm-chart/templates/configMaps.yaml

```diff
@@ -349,3 +349,320 @@ data:
+    GOOGLE_ANALYTICS_CODE = os.getenv("GOOGLE_ANALYTICS_CODE")
+    GOOGLE_MAPS_API_KEY = os.getenv("GOOGLE_MAPS_API_KEY")
+    GOOGLE_APPLICATION_CREDENTIALS_FILEPATH = "/google-cloud-secret/BackupManagerKey.json"
+    MIXPANEL_CODE = os.getenv("MIXPANEL_CODE")
+
+    HOTJAR_ID = os.getenv("HOTJAR_ID")
+
+    AWS_ACCESS_KEY = os.getenv("AWS_ACCESS_KEY")
+    AWS_SECRET_KEY = os.getenv("AWS_SECRET_KEY")
+    S3_BUCKET = os.getenv("S3_BUCKET")
+
+    NATIONBUILDER = True
+    NATIONBUILDER_SLUG = "sefaria"
+    NATIONBUILDER_TOKEN = os.getenv("NATIONBUILDER_TOKEN")
+    NATIONBUILDER_CLIENT_ID = os.getenv("NATIONBUILDER_CLIENT_ID")
+    NATIONBUILDER_CLIENT_SECRET = os.getenv("NATIONBUILDER_CLIENT_SECRET")
+
+    DISABLE_INDEX_SAVE = False
+
+    # Turns off search autocomplete suggestions, which are reinitialized on every server reload
+    # which can be annoying for local development.
+    DISABLE_AUTOCOMPLETER = False
+
+    PARTNER_GROUP_EMAIL_PATTERN_LOOKUP_FILE = "/school-lookup-data/schools.tsv"
+
+
+    MAILCHIMP = True
+    MAILCHIMP_API_KEY = os.getenv("MAILCHIMP_API_KEY")
+    MAILCHIMP_ANNOUNCE_ID = os.getenv("MAILCHIMP_ANNOUNCE_ID")
+    MAILCHIMP_WEBHOOK_KEY = os.getenv("MAILCHIMP_WEBHOOK_KEY")
+
+    USE_VARNISH = True
+    FRONT_END_URL = os.getenv("FRONT_END_URL")
+    VARNISH_ADM_ADDR = "{}:6082".format(varnishHost)
+    VARNISH_HOST = varnishHost
+    VARNISH_FRNT_PORT = 8040
+    VARNISH_SECRET = "/varnish-secret/varnish-secret"
+    USE_VARNISH_ESI = False
+
+    RECAPTCHA_PUBLIC_KEY = os.getenv("RECAPTCHA_PUBLIC_KEY")
+    RECAPTCHA_PRIVATE_KEY = os.getenv("RECAPTCHA_PRIVATE_KEY")
+    NOCAPTCHA = True
+
+    # Simple JWT
+    SIMPLE_JWT = {
+        'ACCESS_TOKEN_LIFETIME': timedelta(days=1),
+        'REFRESH_TOKEN_LIFETIME': timedelta(days=90),
+        'ROTATE_REFRESH_TOKENS': True,
+        'SIGNING_KEY': os.getenv("SIMPLE_JWT_SIGNING_KEY"),
+    }
+
+    MOBILE_APP_KEY = os.getenv("MOBILE_APP_KEY")
+
+    LOGGING = {
+        'version': 1,
+        'disable_existing_loggers': False,
+        'formatters': {
+            "json_formatter": {
+                "()": structlog.stdlib.ProcessorFormatter,
+                "processor": structlog.processors.JSONRenderer(),
+            },
+        },
+        'handlers': {
+            'default': {
+                "class": "logging.StreamHandler",
+                "formatter": "json_formatter",
+            },
+        },
+        'loggers': {
+            '': {
+                'handlers': ['default'],
+                'propagate': False,
+            },
+            'django': {
+                'handlers': ['default'],
+                'propagate': False,
+            },
+            'django.request': {
+                'handlers': ['default'],
+                'propagate': False,
+            },
+        }
+    }
+
+    # TODO: Make the logging format more configurable based on specific objects being defined.
```

Comment:
I think we can put the entire JSON value into configmap key

Reply:
The logging objects change based on environment and there are several others that need to be added. When discussed with Brendan he said for a first pass we can go ahead with the prod configuration as default and then make others configurable within the python script based on environment variables set.